sites/maas.io.yaml

```diff
@@ -13,11 +13,19 @@ env:
 
 extraHosts:
   - domain: docs.maas.io
+  - domain: maas.ubuntu.com
```

Comment:
Do we have TLS secretes set for this domains?

Reply:
this is on IS side. It's in the comment that I give when I announce it's ready